### PR TITLE
Codepush/configuration tests

### PR DESCRIPTION
**Pull request by sergey-akhalkov**

### CODE REVIEW

File: sdk/appcenter-codepush/src/test/java/com/microsoft/appcenter/codepush/CodePushConfigurationTest.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({ AppCenterLog.class })
+public class CodePushConfigurationTest {
+    @Before
+    public void setUp() {
+        mockStatic(AppCenterLog.class);
+    }
+
+
```
Comment (MatkovIvan) [inline sdk/appcenter-codepush/src/test/java/com/microsoft/appcenter/codepush/CodePushConfigurationTest.java]: Remove one empty line

File: sdk/appcenter-codepush/src/test/java/com/microsoft/appcenter/codepush/CodePushConfigurationTest.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    @Test
+    public void correctConfigurationTest() {
+        CodePushConfiguration correctConfig = new CodePushConfiguration();
+        correctConfig.setAppVersion("1.0.0")
+                .setClientUniqueId("clientUniqueId")
+                .setDeploymentKey("deploymentKey")
+                .setPackageHash("packageHash")
+                .setServerUrl("serverUrl");
+
```
Comment (MatkovIvan) [inline sdk/appcenter-codepush/src/test/java/com/microsoft/appcenter/codepush/CodePushConfigurationTest.java]: Remove empty line or add a comment

File: sdk/appcenter-codepush/src/test/java/com/microsoft/appcenter/codepush/CodePushConfigurationTest.java (hunk 1 above)
Comment (sergey-akhalkov) [reply sdk/appcenter-codepush/src/test/java/com/microsoft/appcenter/codepush/CodePushConfigurationTest.java]: Fixed, thanks

File: sdk/appcenter-codepush/src/test/java/com/microsoft/appcenter/codepush/CodePushConfigurationTest.java (hunk 2 above)
Comment (sergey-akhalkov) [reply sdk/appcenter-codepush/src/test/java/com/microsoft/appcenter/codepush/CodePushConfigurationTest.java]: Fixed, thanks

File: sdk/appcenter-codepush/src/test/java/com/microsoft/appcenter/codepush/CodePushConfigurationTest.java [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        CodePushConfiguration wrongConfig = new CodePushConfiguration();
+        wrongConfig.setAppVersion(null)
+                .setClientUniqueId(null)
+                .setDeploymentKey(null)
+                .setServerUrl(null);
+        assertEquals(null, wrongConfig.getAppVersion());
+        assertEquals(null, wrongConfig.getClientUniqueId());
+        assertEquals(null, wrongConfig.getDeploymentKey());
+        assertEquals(null, wrongConfig.getServerUrl());
+
```
Comment (MatkovIvan) [inline sdk/appcenter-codepush/src/test/java/com/microsoft/appcenter/codepush/CodePushConfigurationTest.java]: Add comment or remove

**Pull request merged**